src/applications/edu-benefits/0994/components/PaymentReviewView.jsx

```diff
@@ -16,16 +16,20 @@ export class PaymentReviewView extends React.Component {
     const { formData, name } = this.state;
     let value = formData;
     if (formData === undefined) {
-      // Use prefill data
-      const propName = `bank${name
-        .substring(0, 1)
-        .toUpperCase()}${name.substring(1)}`;
-      const prefillBankAccount = _.get(
-        this.props.data,
-        'prefillBankAccount',
-        {},
-      );
-      value = _.get(prefillBankAccount, propName, '');
+      const viewBankAccount = _.get(this.props.data, 'view:bankAccount', {});
+
+      if (!hasNewBankInformation(_.get(viewBankAccount, 'bankAccount', {}))) {
```

Comment:
maybe consolidate this into one `_.get(this.props.data, 'view:bankAccount.bankAccount', {})`